Flake TestGetSetFileValues

<!--
  Issues without logs and details are more complicated to fix.
  Please help us by filling the template below!
-->

### Expected behavior
Test should not rely on order.
### Actual behavior
```
--- FAIL: TestGetSetFileValues (0.00s)
    --- FAIL: TestGetSetFileValues/multiple_value (0.00s)
        helm_test.go:777: []string differ (-got, +want):   []string{
              	"--set-file",
            - 	"another_file=path/to/another",
            + 	"multiline_text=path/to/textfile",
              	"--set-file",
            - 	"multiline_text=path/to/textfile",
            + 	"another_file=path/to/another",
              }
time="2019-09-24T12:53:07-07:00" level=warning msg="missing.yaml did not match any file"
time="2019-09-24T12:53:07-07:00" level=info msg="refusing to deploy/delete non {json, yaml} file empty.ignored"
time="2019-09-24T12:53:07-07:00" level=info msg="If you still wish to deploy this file, please specify it directly, outside a glob pattern."
time="2019-09-24T12:53:07-07:00" level=info msg="error decoding parsed yaml: couldn't get version/kind; json parse error: json: cannot unmarshal string into Go value of type struct { APIVersion string \"json:\\\"apiVersion,omitempty\\\"\"; Kind string \"json:\\\"kind,omitempty\\\"\" }"
FAIL
```
### Information

- Skaffold version: version...
- Operating system: ...
- Contents of skaffold.yaml:

```yaml
<paste your skaffold.yaml here>
```

### Steps to reproduce the behavior

1. a clonable repository with the sample skaffold project
2. `skaffold <command>`
3. ...

--- a/pkg/skaffold/deploy/helm_test.go
+++ b/pkg/skaffold/deploy/helm_test.go
@@ -25,6 +25,7 @@ import (
 	"os"
 	"os/exec"
 	"runtime"
+	"sort"
 	"strings"
 	"testing"
 
@@ -774,6 +775,8 @@ func TestGetSetFileValues(t *testing.T) {
 		testutil.Run(t, test.description, func(t *testutil.T) {
 			inMap := map[string]bool{}
 			actual := generateGetFilesArgs(test.files, inMap)
+			sort.Strings(test.expected)
+			sort.Strings(actual)
 			t.CheckDeepEqual(test.expected, actual)
 			t.CheckDeepEqual(test.expectedMap, inMap)
 		})

